nginx-ingress-controller-image/root/etc/nginx/opentracing.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
I assume this is not error right? This empty json file is requierd.

Reply:
Yes, this empty json file is required for opentracing integration.